### PR TITLE
Examples: Clean up.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

Some code in `examples/jsm/objects` still needed to be converted to string templates.

### CODE REVIEW

**Comment by Mugen87** [comment]
The changes related to formatting are due to #21677.

File: examples/js/animation/CCDIKSolver.js [hunk 1]
```diff
@@ -45,9 +45,9 @@
 	class CCDIKSolver {
 
 		/**
-	 * @param {THREE.SkinnedMesh} mesh
-	 * @param {Array<Object>} iks
-	 */
+   * @param {THREE.SkinnedMesh} mesh
```
Comment (Mugen87) [inline examples/js/animation/CCDIKSolver.js:48]: @marcofugaro Do you know why these line are changed? Does this not happen if I manually let eslint format the file.

Comment (marcofugaro) [reply examples/js/animation/CCDIKSolver.js:48]: @Mugen87 so basically eslint does not touch indentation in comment blocks.
In the `npm run build-examples` script there is babel who rewrites everything and converts tabs to spaces, and then eslint can't convert them back.

Running eslint from the editor, it leaves the indentation in the comment blocks as is, so with tabs.

Comment (Mugen87) [reply examples/js/animation/CCDIKSolver.js:48]: Okay. I think this is a minor issue and can be ignored. The canonical files are in `examples/jsm` and only they have to "look good". For `examples/js` it's sufficient that the examples are functional.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**